docs/api-docs/payments/payments-api-overview.md

```diff
@@ -382,19 +265,20 @@ lineNumbers: true
 }
 ```
 
-If the purchase was successful it will return a status of success. The order is then automatically moved to an Awaiting Fulfillment status. If you get a different response, see [Error Codes](#error-codes) for troubleshooting.
+If the purchase was successful, the response returns a status of success. The order is then automatically moved to an Awaiting Fulfillment status. If you get a different response, see [Error codes](#error-codes) for troubleshooting.
 
-### Storing credit cards
+### Storing credit cards and paypal accounts
 
-The payments API allows developers to store a credit card while processing a credit card payment.
+The payments API allows developers to store a credit card or paypal account while processing a payment.
```

Comment:
```suggestion
The payments API allows developers to store a credit card while processing a payment.
```
We don't allow paypal account payments via API. They have to be stored via storefront checkout. Once stored, they can be used though